src/Microsoft.ML.FastTree/QuantileStatistics.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.ML
 {
-    public sealed class QuantileStatistics : IQuantileDistribution<float>
+    internal sealed class QuantileStatistics : IQuantileDistribution<float>
```

Comment:
>QuantileStatistics [](start = 26, length = 18)

The IQuantileDistribution interface seems to have just this one implementation,
which only uses the GetQuantile() method. Can this be defined instead as a private class inside FastForestRegressionModelParameters?

Reply:
That would probably be positive. Then we can delete that internal `IQuantileDistribution` interface altogether. A quick and easy improvement in case we need to update this PR again.

---
In reply to: [258227442](https://github.com/dotnet/machinelearning/pull/2511#discussion_r258227442) [](ancestors = 258227442)